components/Footer/Footer.tsx

```diff
@@ -155,7 +157,7 @@ const EmailSignup = () => (
 
 export const Footer = () => (
     <div className={styles.banner}>
-        <EmailSignup />
+        <Banner />
```

Comment:
**Nickpick**: 
```suggestion
        <FooterBanner />
``` Can be nice to prefix names to match the file it's being used in, also less confusing for developers looking for components.